### PR TITLE
switching to use d2l-icon instead of custom iconset

### PR DESCRIPTION
**Pull request by dlockhart**
A couple reasons for this change:
1) We'd like to deprecate the use of custom `iron-iconset-svg`s, as using them has become a performance problem and they're not supported in Lit. The usage here seems unnecessary, since I can't tell the different between the chevrons it was using and a shrunken down version of our tier1 chevrons.
2) This darkens the icon slightly to match the new WCAG 2.1 colours.

Before the change:
Screen Shot 2019-06-05 at 3 04 34 PM

After the change:
Screen Shot 2019-06-05 at 3 02 36 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,8 +20,6 @@
     "@babel/polyfill": "^7.0.0",
     "@polymer/iron-component-page": "^3.0.0-pre.18",
     "@polymer/iron-demo-helpers": "^3.0.0-pre.18",
-    "@polymer/iron-icon": "^3.0.0-pre.18",
-    "@polymer/iron-iconset-svg": "^3.0.0-pre.18",
```
Comment (dlockhart) [inline package.json]: Interestingly because this was only a devDependency before, it was relying on others to include `iron-icon` and `iron-iconset-svg`.

**Comment by dlockhart** [comment]
Oh dear. I'll try adding one... what are the chances it passes?

**Comment by svanherk** [comment]
> Oh dear. I'll try adding one... what are the chances it passes?

There's no siren stuff or heavy javascript for IE11 to trip over... I'm gonna go with 75% 🤞

**Comment by dlockhart** [comment]
🎉

**Pull request merged**